deployment/grid/terraform/compute_plane/grafana.tf

```diff
@@ -112,6 +112,7 @@ resource "kubernetes_ingress_v1" "grafana_ingress" {
   depends_on = [
     #kubernetes_namespace.grafana,
     module.eks_blueprints_kubernetes_addons,
+    module.eks_blueprints_kubernetes_addons_for_alb
```

Comment:
This would complicate the building of the dependency tree massively - would be much simpler to keep everything in one self contained module.